packages/core/admin/_internal/cli/commands/build.ts

```diff
@@ -21,8 +23,37 @@ const build = async (options: BuildCLIOptions) => {
       );
     }
 
+    if (options.bundler !== 'webpack') {
+      options.logger.log(
+        boxen(
+          `Using ${chalk.bold(
+            chalk.underline(options.bundler)
+          )} as a bundler is considered experimental, use at your own risk. If you do experience bugs, open a new issue on Github – https://github.com/strapi/strapi/issues/new?template=BUG_REPORT.md`,
+          {
+            title: 'Warning',
+            padding: 1,
+            margin: 1,
+            align: 'center',
+            borderColor: 'yellow',
+            borderStyle: 'bold',
+          }
+        )
+      );
+    }
+
     const envSourceMaps = process.env.STRAPI_ENFORCE_SOURCEMAPS === 'true';
```

Comment:
Not sure I understand why we need that env var as the cli has an option ? was it for testing or sth ?

Reply:
Backwards compat – we originally had it before, but we can remove it in V5.